travis.sh

```diff
@@ -248,40 +236,26 @@ function test_workspace() {
    echo -e $(colorize GREEN Testing Workspace)
    travis_run_simple --title "Sourcing newly built install space" source install/setup.bash
 
-   # Consider TEST_BLACKLIST
-   TEST_BLACKLIST=$(unify_list " ,;" $TEST_BLACKLIST)
```

Comment:
Instead of removing this it should be implemented with `colcon --skip-packages` or by simply moving `COLCON_IGNORE` files to the package paths.

Reply:
done